alacritty/src/display/wayland_theme.rs

```diff
@@ -28,12 +28,12 @@ impl AlacrittyWaylandTheme {
         dim_foreground.a = INACTIVE_OPACITY;
 
         Self {
-            foreground,
             background,
+            foreground,
```

Comment:
Can you please change the struct definition here? I don't like background before foreground.